pkg/resolve/selector.go

```diff
@@ -47,6 +47,11 @@ func MatchesSelector(doc *yaml.Node, selector labels.Selector) (bool, error) {
 }
 
 func docKind(doc *yaml.Node) (string, error) {
+	// Null nodes will fail the check below, so simply ignore them.
+	if doc.Tag == "!!null" {
+		return "", nil
+	}
+
```

Comment:
I'd move this up a level to the `MatchesSelector` func. Then you can just return `false` there

Reply:
I was thinking about that, but decided not to because `docKind` is called from both `MatchesSelector` and `listMatchesSelector`. I'm not sure why this would come up for `listMatchesSelector`, but wanted to be consistent just in case.